content/config.yml

```diff
@@ -477,6 +477,32 @@ nav:
         url: '/contact/'
 
 
+susy_nav:
+  - text: 'susysite-articles'
+    display_text: 'articles'
+    url: '/susy/articles/'
+  - text: 'susysite-sites'
+    display_text: 'live sites'
+    url: '/susy/sites/'
+  - text: 'support'
+    subnav:
+      - text: 'docs'
+        display_text: 'reference documentation »'
```

Comment:
The double arrows don't indicate leaving the site anywhere else. maybe we should use an icon instead? or nothing?